datadriven/Cargo.toml

```diff
@@ -7,10 +7,10 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = "1.0.31"
-regex = "1.3.9"
+anyhow = "1.0.40"
+regex = "1"
 lazy_static = "1.4.0"
-slog = "2.5.2"
-slog-term = "2.6.0"
-slog-async = "2.5.0"
+slog = "2.7.0"
```

Comment:
Better keep it as it is. Cargo will try to use latest version when possible.

Reply:
Just to unify the dependency version, please ignore this

Reply:
Then we can keep using 2.2 as the library does. A smaller version leaves more flexibility for downstream to choose a suitable version.